### ISSUE TITLE
Unable to remove ISO when snapshot exists

### ISSUE THREAD

**Issue by mariusvw** [pr1ntf/iohyve#163]
Our server backup system creates recursive snapshots causing the rmiso function to fail.

```
# iohyve rmiso CentOS-7-x86_64-DVD-1511.iso
Deleting CentOS-7-x86_64-DVD-1511.iso...
cannot destroy 'zroot/iohyve/ISO/CentOS-7-x86_64-DVD-1511.iso': filesystem has children
use '-r' to destroy the following datasets:
zroot/iohyve/ISO/CentOS-7-x86_64-DVD-1511.iso@io_zroot-20160504124501
```

A recursive delete just like when you delete an guest resolves the issue.
For now the workaround is removing it directly with the zfs command.

**Comment by pr1ntf** [comment]
What version are you using? This should be fixed in `v0.7.5`.
[Code](https://github.com/pr1ntf/iohyve/blob/master/lib/ioh-iso#L45) in question.

**Comment by pr1ntf** [comment]
I'm closing out this issue for now, however I will reopen if you are on the correct version and still seeing the issue. 

Note that `v0.7.5` is not only on GitHub, but also in the ports tree.
